src/main/groovy/com/avioconsulting/mule/linter/model/Application.groovy

```diff
@@ -2,23 +2,34 @@ package com.avioconsulting.mule.linter.model
 
 class Application {
 
+    static final String APPLICATION_DOES_NOT_EXIST = 'Application directory does not exists.'
+    static final String POM_FILE = 'pom.xml'
+
     File applicationPath
+    List<ProjectFile> files = []
 
     Application(File applicationPath) {
         this.applicationPath = applicationPath
+        if (!this.applicationPath.exists()) {
+            throw new FileNotFoundException(applicationPath.absolutePath, APPLICATION_DOES_NOT_EXIST)
```

Comment:
The 2 argument constructor is private, should be using the 1 argument public constructor

Reply:
changed to use 1 arg constructor.